skaffold.yaml

```diff
@@ -41,4 +41,4 @@ profiles:
               podLabels: { k8s-app: metrics-server }
               containerPort: 10250
             valuesFiles:
-              - charts/metrics-server/ci/ci-values.yaml
```

Comment:
What with this file?

Reply:
When deploying the chart, parameters will be used from this file.
But this file is shared with the Approve GH Workflows test and will affect each other.
Now independent it.